beacon_node/http_api/tests/interactive_tests.rs

```diff
@@ -1050,22 +1053,87 @@ async fn proposer_duties_with_gossip_tolerance() {
     );
 }
 
+// Test that a request to `lighthouse/custody/backfill` succeeds by verifying that `CustodyContext` and `DataColumnCustodyInfo`
+// have been updated with the correct values.
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn lighthouse_restart_custody_backfill() {
```

Comment:
We should add the condition to skip pre fulu forks to avoid running this on prior forks.

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/8339/commits/580b584a8f0838906171a421c3b404ae8a905e07